.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
let's delete this and add it to the .gitignore

Reply:
done